src/makerSelection.ts

```diff
@@ -0,0 +1,65 @@
+import {
+	BN,
+	convertToNumber,
+	divCeil,
+	DLOBNode,
+	ZERO,
+} from '@drift-labs/sdk';
+import { MAX_MAKERS_PER_FILL } from './bots/filler';
+
+const PROBABILITY_PRECISION = new BN(1000);
+
+export function selectMakers(makerNodes: DLOBNode[]): DLOBNode[] {
+	const selectedMakers = [];
```

Comment:
should this be a Set, to avoid duplicate makers? or is makerNodes expected to be unique

Reply:
good point